### PR TITLE
feat(more updates): About me, contact links, rainbow time! 🌈 

### PR DESCRIPTION
**Pull request by wtfluckey**

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -99,6 +99,16 @@ p:last-of-type {
 /**
  * Links
  */
+
+button, a {
```
Comment (leongersing) [inline _sass/_base.scss:103]: I understood that reference

File: _sass/_components.scss [hunk 2]
```diff
@@ -1,37 +1,27 @@
...
 .border-footer {
   @include border;
   padding-bottom: 10px;
 }
 
+body.rainbow-time .border-footer {
+  @include border-rainbow;
+  padding-bottom: 10px;
+}
+
```
Comment (wtfluckey) [inline _sass/_components.scss:20]: TODO (next PR): refactor these `body.rainbow` styles to theme things better

File: _sass/_components.scss [hunk 3]
```diff
@@ -1,37 +1,27 @@
...
-  background-size: 200% 200%;
-  animation: slide 5s linear infinite;
-}
-
 .border-header {
   @include border;
   padding-top: 10px;
 }
 
+body.rainbow-time .border-header {
```
Comment (leongersing) [inline _sass/_components.scss:6]: 🌈

File: _sass/_components.scss [hunk 4]
```diff
@@ -45,21 +35,58 @@
   }
 }
 
-.test {
+body.rainbow-time .title {
```
Comment (leongersing) [inline _sass/_components.scss:38]: /play tmyk

File: _sass/_mixins.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
+@keyframes slide {
```
Comment (leongersing) [inline _sass/_mixins.scss:1]: :fire:

File: index.html [hunk 6]
```diff
@@ -1,13 +1,62 @@
...
+    <button>
+      <span class="icon rainbow tooltip" id="cue-rainbow-time" data-tooltip="Toggle rainbow time! 🌈">
+        {% include rainbow.svg %}
+      </span>
+    </button>
+  </div>
 </div>
 
+<script>
+  document.getElementById('cue-rainbow-time').addEventListener('click', function () {
```
Comment (leongersing) [inline index.html:58]: SCRIPT TIME

**Review by leongersing — APPROVED** [review]
It's so good you should SHIP IT NOW!!! 

RAINBOWS

File: index.html [hunk 7]
```diff
@@ -1,13 +1,62 @@
...
+    great.</p>
   </div>
 
-  <a href="{{ site.baseurl }}/assets/AlyFluckey-resume-2024.pdf" download>Download my resume</a>
+  <div class="contact">
+    <a href="mailto:fluckeya@gmail.com?subject=Hello!%20%F0%9F%91%8B&body=Please%20write%20your%20message%20here">
+      <span class="icon tooltip" data-tooltip="Email me!">
+        {% include email.svg %}
+      </span>
+    </a>
```
Comment (wtfluckey) [inline index.html:29]: @leongersing 

email?

File: index.html (hunk 6 above)
Comment (wtfluckey) [reply index.html:58]: giphy-1

File: index.html (hunk 7 above)
Comment (leongersing) [reply index.html:29]: like a person

File: _sass/_base.scss (hunk 1 above)
Comment (wtfluckey) [reply _sass/_base.scss:103]: giphy

**Pull request merged**